code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,255 @@
+	var/obj/item/slimecross/warping/extract
+	var/pickuptime = 20 //2 seconds
+
+/obj/effect/slimerune/attack_hand(mob/living/user)
+	. = ..()
+	var/mob/living/L = user
+	if(!istype(user))
+		return
+	if(.)
+		return
+	to_chat(user, "<span class='warning'>You begin to gather [src] up off the ground.</span>")
+	if(do_after(user, pickuptime, src))
+		to_chat(user, "<span class='notice'>You collect [src] into a pile, and it reforms into [extract]!</span>")
+		extract.forceMove(loc)
+		on_remove()
+		qdel(src)
+		L.put_in_hands(extract)
+
+/obj/effect/slimerune/Initialize(mapload)
+	. = ..()
+	on_place()
+
+/obj/effect/slimerune/Destroy()
+	STOP_PROCESSING(SSobj,src)
+	return ..()
+
+/obj/effect/slimerune/proc/on_place()
+	START_PROCESSING(SSobj,src)
+	return
+
+/obj/effect/slimerune/proc/on_remove()
+	return
+
+/obj/item/slimecross/warping/grey/
+	colour = "grey"
+	rune_path = /obj/effect/slimerune/grey
+	effect_desc = "Forms a recoverable rune that absorbs slime extracts to produce baby slimes."
+	var/num_absorbed = 0
+
+/obj/item/slimecross/warping/orange
+	colour = "orange"
+	rune_path = /obj/effect/slimerune/orange
+	effect_desc = "Forms a recoverable rune that functions as a fancy, magical bonfire."
+
+/obj/item/slimecross/warping/purple
+	colour = "purple"
+	rune_path = /obj/effect/slimerune/purple
+	effect_desc = "Forms a recoverable rune that converts cloth and plastic to medical supplies."
+
+/obj/item/slimecross/warping/blue
+	colour = "blue"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that douses the area it is on with water."
+
+/obj/item/slimecross/warping/metal
+	colour = "metal"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that functions as a transparent wall."
+
+/obj/item/slimecross/warping/yellow
+	colour = "yellow"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that drains batteries on the tile to fuel the area's APC."
+
+/obj/item/slimecross/warping/darkpurple
+	colour = "dark purple"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that grows crystallized plasma over time. It's too fragile to make sheets out of..."
+
+/obj/item/slimecross/warping/darkblue
+	colour = "dark blue"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that lowers the body temperature of creatures that enter its circle."
+
+/obj/item/slimecross/warping/silver
+	colour = "silver"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that absorbs food to feed those who cross its path."
+
+/obj/item/slimecross/warping/bluespace
+	colour = "bluespace"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that links a <b>wormhole satchel</b> with the space above it."
+
+/obj/item/slimecross/warping/sepia
+	colour = "sepia"
+	rune_path = /obj/effect/slimerune
+	effect_desc = "Forms a recoverable rune that holds any living creature in its influence under stasis."
```

Comment:
Statis as in what statis beds do or statis as in what timestop does? please god tell me it's the latter because i want to make a time jail.

Reply:
As in what stasis beds do, unfortunately.

Reply:
Aw.